src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -83,9 +84,17 @@ public static void generateReposReport(List<RepoConfiguration> configs, String o
                 ? earliestSinceDate : cliSinceDate;
 
         FileUtil.writeJsonFile(
-                new SummaryReportJson(configs, generationDate, reportSinceDate, untilDate, RepoSense.getVersion()),
+                new SummaryJson(configs, generationDate, reportSinceDate, untilDate, RepoSense.getVersion()),
                 getSummaryResultPath(outputPath));
         logger.info(String.format(MESSAGE_REPORT_GENERATED, outputPath));
+
+        List<Path> reportFoldersAndFiles = new ArrayList<>();
+        for (RepoConfiguration config : configs) {
+            reportFoldersAndFiles.add(
+                    Paths.get(outputPath + File.separator + config.getDisplayName()).toAbsolutePath());
```

Comment:
Still not really the ideal solution because this assumes that the files were generated which might not be the case, but we can settle that in another issue. 

The ideal solution should the methods, that generate the file e.g. FileUtil.writeJsonFile, return the path of the files that were successfully generated. Only then, `generateReposReport` should consolidate them one by one and return it to the caller.

Can you please open an issue to target this problem?

Reply:
Opened up a new issue on this.